### ISSUE TITLE
what going to be our preferred tech stack?

### ISSUE THREAD

**Issue by codedawi** [opensource-covid19/community#11]
We should try to stick to a fairly consistant tech stack to allow for more collaboration.

**Comment by codedawi** [comment]
Id prefer we use `python` for much of our projects to keep the barrier of entry fairly low for new contributors. Also, `python` is widely used across the data science community. With that being said, I'd recommend us using `flask`, `django`, or `fastapi` for our web apps/apis.

**Comment by ColinFletch** [comment]
Yeah I'd second that, the barrier to entry for python is lower in general, and especially if we use a framework such as flask or django.
